api/envoy/api/v2/listener/listener.proto

```diff
@@ -204,7 +204,18 @@ message FilterChain {
   string name = 7;
 }
 
+// Specifies the match criteria. All of the criteria needs to be satisfied.
+message ListenerFilterMatch {
+  // If it is not empty, the criteria is satisfied if the destination port of the connection is
+  // contained.
+  repeated uint32 destination_ports = 1
+      [(validate.rules).repeated = {items {uint32 {lte: 65535 gte: 1}}}];
+}
+
+// [#next-free-field: 6]
 message ListenerFilter {
+  reserved 5;
```

Comment:
? (I think per your comment below you intent to add new fields later, but just do it later, this can be removed now)

Reply:
it's gone since `not_match` is introduced